core/backend/src/CustomViewState3dCreator.ts

```diff
@@ -61,11 +61,18 @@ export class CustomViewState3dCreator {
     const modelIds = new Set(modelIdsList);
     Logger.logInfo(loggerCategory, "Starting getModelExtents query.");
     for (const id of modelIds) {
+      const modelExtentsStopWatch = new StopWatch("getModelExtents query", false);
       try {
         await new Promise((resolve) => setImmediate(resolve)); // Free up main thread temporarily. Ideally we get queryModelExtents off the main thread and do not need to do this.
+        Logger.logInfo(loggerCategory, "Starting getModelExtents query.");
```

Comment:
Shouldn't you log the model Id so we can investigate specific models that take too long?

Reply:
Changed it